integration/integration_test.go

```diff
@@ -4749,15 +4749,15 @@ func testX11Forwarding(t *testing.T, suite *integrationTestSuite) {
 							assert.Eventually(t, func() bool {
 								output, err := os.ReadFile(tmpFile.Name())
 								if err == nil && len(output) != 0 {
-									display = strings.TrimSpace(string(output))
+									display <- strings.TrimSpace(string(output))
```

Comment:
I think a buffer of 1 should be okay, since as soon as this write occurs the entire `require.EventuallyWithT` should complete, but would appreciate more 👀

Reply:
Could make it a non-blocking channel write that just discards the value if there's something in the channel already?